sdk/src/native_loader.rs

```diff
@@ -1,13 +1,46 @@
 use crate::{account::Account, hash::Hash};
+use num_derive::FromPrimitive;
 
 crate::declare_id!("NativeLoader1111111111111111111111111111111");
 
+#[derive(Debug, Clone, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct Info {
+    pub kind: Kind,
+    pub name: String,
+}
+#[derive(Debug, Clone, Copy, Deserialize, Eq, FromPrimitive, Hash, PartialEq, Serialize)]
+pub enum Kind {
```

Comment:
The name of this file, and then the `Kind` enum, trips me up.   This file appears to be about `native_loader`s and there are two `Kind`s of them: `Program` and `Loader`.


Perhaps rename the file to `native_program.rs` and then there are two `Kinds` of native programs: a normal `Kind::Program`, and `Kind::Loader` that can load other problems.   Does that seem to make sense?

Reply:
Kinda, This file is the sdk (public) component of the `native_loader`, thus is contains the program id definition and the helper to create a loadable account.  Are you suggesting that we move `info` and `kind` (or maybe just `kind`) out to a new file called `native_program.rs`?

Reply:
Maybe I'm just unclear on when I'd use `Kind::Program` vs `Kind::Loaader`

Reply:
They are by the macros in `sdk/src/native_loader.rs` and in the `NativeLoader` implementation in `runtime/src/native_loader.rs`.  For the SDK consumer, they are wrapped by the macros.  This brings me to a larger question, this file is one that should probably not be in the "SDK" and instead should be in a "common" (yuck) spot in the tree.